tests/unit/cocoon/test_properties.py

```diff
@@ -121,16 +131,18 @@ def test_check_property_definitions_exist_in_scope_single(
 
         :return: None
         """
-
-        (
-            property_existence,
-            property_type,
-        ) = cocoon.properties.check_property_definitions_exist_in_scope_single(
-            api_factory=self.api_factory, property_key=property_key
-        )
-
-        self.assertEqual(property_existence, expected_outcomes[0])
-        self.assertEqual(property_type, expected_outcomes[1])
+        try:
+            (
+                property_existence,
+                property_type,
+            ) = cocoon.properties.check_property_definitions_exist_in_scope_single(
+                api_factory=self.api_factory, property_key=property_key
+            )
+
+            self.assertEqual(property_existence, expected_outcomes[0])
+            self.assertEqual(property_type, expected_outcomes[1])
+        except:
```

Comment:
I recommend making this except `ApiException as api_exception` to ensure you are only catching an ApiException and not something you didn't expect.

Reply:
This should be fixed now